troi/filters.py

```diff
@@ -377,14 +386,17 @@ class NeverListenedFilterElement(troi.Element):
         Remove/keep only recordings if they have not/been listened to.
     '''
 
-    def __init__(self, remove_unlistened=True):
+    MIN_ITEMS_REQUIRED = 5
```

Comment:
Just checking that this is expected to be defined again.
I see self.MIN_ITEMS_REQUIRED so I suppose it is a class const rather than a global one.